### PR TITLE
System auth rf now used the credentials.

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -398,13 +398,13 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self.parent_cluster = parent_cluster  # reference to the Cluster object that the node belongs to
         self.logdir = os.path.join(base_logdir, self.name)
         makedirs(self.logdir)
+        self.log = SDCMAdapter(LOGGER, extra={'prefix': str(self)})
```
Comment (amoskong) [inline sdcm/cluster.py]: Why do you move this line?

Comment (aleksbykov) [reply sdcm/cluster.py]: I think, we should initiate self.log on earlier, once main properties are defined

Comment (amoskong) [reply sdcm/cluster.py]: ok

File: sdcm/tester.py [hunk 2]
```diff
@@ -290,6 +280,21 @@ def setUp(self):
 
         self.db_cluster.validate_seeds_on_all_nodes()
 
+    def set_system_auth_rf(self):
+        # change RF of system_auth
+        system_auth_rf = self.params.get('system_auth_rf', default=3)
```
Comment (bentsi) [inline sdcm/tester.py]: we discussed, no params defaults in the code!!!!!

**Comment by bentsi** [comment]
please check why unit tests failed

File: sdcm/tester.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/tester.py]: Adding to test_default.yaml

**Comment by aleksbykov** [comment]
Run unit tests. All passed.

**Comment by bentsi** [comment]
please resolve conflicts

**Comment by bentsi** [comment]
Let's backport when stable on master

**Pull request merged**